static/js/main.js

```diff
@@ -375,6 +397,14 @@ function takeScreenshot(source, x, y, width, height, format = 'image/jpeg') {
     return canvas.toDataURL(format);
 }
 
+function deleteScreenshot(screenshot_id) {
+    // Find screenshot area to remove.
+    let screenshot_area = document.querySelector(`.screenshot[data-id="${screenshot_id}"]`);
+    screenshot_area.remove();
```

Comment:
I would merge this to a one-liner.
Since `area` is not used elsewhere and we already established the convention of operating directly off a `querySelector()`, it should be fine.